agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/Configuration.java

```diff
@@ -443,8 +442,6 @@ public void validate(ProcessorType processorType, IncludeExclude includeExclude)
                     validAttributeProcessorIncludeExclude(includeExclude);
                     break;
                 case log:
-                    validateLogProcessorIncludeExclude(includeExclude);
-                    break;
```

Comment:
I think it makes sense to keep this, even if it only validates non-empty attributes, in order to keep friendly message specific to log processor